jest.config.ts

```diff
@@ -125,9 +125,9 @@ export default {
   // runner: "jest-runner",
 
   // The paths to modules that run some code to configure or set up the testing environment before each test
-  setupFiles: [
-    "./setup-jest.js",
-  ],
+  // setupFiles: [
+  //   "./setup-jest.js",
+  // ],
```

Comment:
Please remove comments

Reply:
Do you want me to remove all comments in the file? They were kept there before, I guess for easier configuration, so I decided not to remove them but we can get rid of them.

Reply:
for me it can stay

Reply:
I would honestly keep it for easier future configuration if anyone needed to change anything here